entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,21 @@
+// xxx.js
+// xxx.js
+import prompt from '@system.prompt';
+export default {
+    data: {
+        weather: '/common/images/partly sunny.png',
+        one: '/common/images/1.png',
+        two: '/common/images/2.png',
```

Comment:
1. Date details, please fetch from Date api and display
2. Weather details please fetch from openweather apis and display.